.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -208,6 +208,18 @@ def skip_if_no_python_changes(overrides: Optional[Sequence[str]] = None):
     return "No python changes"
 
 
+def skip_if_no_pyright_requirements_txt_changes():
+    if not is_feature_branch():
+        return None
+
+    if any(
+        Path("pyright", "master", "requirements.txt") in path.parents for path in ChangedFiles.all
```

Comment:
As in the internal PR, we need to check if any `pyright/*/requirements.txt` has changed.